extension/src/store/actions.js

```diff
@@ -269,10 +294,40 @@ export default ({ apollo }) => {
     return wallet.cosmosAddress
   }
 
+  const testSeed = async (store, { networkId, address, seedPhrase }) => {
```

Comment:
remove?

Reply:
is it not needed for extension? I think it is. Let me try

Reply:
As I thougth, it is needed;

![image](https://user-images.githubusercontent.com/40721795/95749200-85409b00-0c9b-11eb-8532-2e94b7a3f748.png)